hacenv.yml

```diff
@@ -1,12 +1,20 @@
-name: transformers
+name: transformers-hf
 dependencies:
     - python=3.8
     - pip
     - pip:
         - torch==1.10.0+cu111.moreh23.6.0
         - moreh-driver==23.6.0
-        - -e ".[dev-torch]"
-
+        - transformers==4.29.2
```

Comment:
I suggest installing `transformers` by source, in editable mode as before (by `-e "."`), instead of pulling from PyPI like this. 
By doing this, you'll have the `transformers` source in your working directory that you can modify & debug easily.

Reply:
good point, let me change this back